src/encoded/__init__.py

```diff
@@ -32,13 +32,29 @@
     json_asset
 )
 from dcicutils.log_utils import set_logging
+from dcicutils.beanstalk_utils import whodaman
```

Comment:
Although used internally here, this amounts to an export of this name from `encoded`. Minimally, I would do this import using `as _whodaman` to emphasize the intent is not that. (You'd have to change a usage farther down, too.)

We really should fix that naming at some point to something respectable.